@xen-orchestra/lite/src/components/pool/dashboard/ramUsage/HostsRamUsage.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <UsageBar :data="data" :n-items="5">
+    <template #header>
+      <span>Hosts</span>
+      <span>Top 5</span>
+    </template>
+  </UsageBar>
+</template>
+
+<script lang="ts" setup>
+import { type ComputedRef, computed, inject } from "vue";
+import UsageBar from "@/components/UsageBar.vue";
+import { formatSize, parseRamUsage } from "@/libs/utils";
+import type { HostStats } from "@/libs/xapi-stats";
+
+const stats: ComputedRef<
+  {
+    name: string;
+    stats?: HostStats;
+  }[]
+> = inject<any>("hostStats", []);
```

Comment:
You should move the typing in replacement of `any` in `inject` (and replace the default value `[]`, which is incompatible with the expected type):

```suggestion
const stats = inject<
  ComputedRef<
    {
      name: string;
      stats?: HostStats;
    }[]
  >
>(
  "hostStats",
  computed(() => [])
);
```

The default value should be compatible with `ComputedRef` instead of an array, because later you call `stats.value.forEach` (which would throw an error if `stats` is an array)